src/components/App.tsx

```diff
@@ -2,14 +2,15 @@ import React from "react";
 import "./App.scss";
 import { Header } from "./navigation/Header";
 import { Route, Routes } from "react-router-dom";
-import { LandingPage } from "./landingPage/LandingPage";
+import { Home } from "./home/Home";
+import { Journey } from "./journey/Journey";
 
 const App: React.FunctionComponent = () => {
   return (
     <>
       <Header />
       <Routes>
-        <Route path="/" element={<LandingPage />} />
+        <Route path="/" element={<Journey />} />
```

Comment:
Is this the correct path for the journey component?

Reply:
Update made